src/TorchSharp/TorchVision/models/AlexNet.cs

```diff
@@ -81,14 +83,19 @@ public AlexNet(int numClasses, float dropout = 0.5f, torch.Device device = null)
                     Dropout(probability: dropout),
                     Linear(256 * 6 * 6, 4096),
                     ReLU(inPlace: true),
-                    Dropout(probability:dropout),
+                    Dropout(probability: dropout),
                     Linear(4096, 4096),
                     ReLU(inPlace: true),
                     Linear(4096, numClasses)
                 );
 
                 RegisterComponents();
 
+                if (!string.IsNullOrEmpty(weights_file)) {
+
+                    this.load(weights_file, skip: skipfc ? new[] { "classifier.6.weight", "classifier.6.bias" } : null);
```

Comment:
>new[] { "classifier.6.weight" [](http://example.com/codeflow?start=59&length=29)

is it possible to have the load method take a span instead of array and avoid the allocation?

Reply:
load() is not a high-frequency method, since it's used just once in a while. I've tried to use 'IList<T>' for most APIs, in order to be as general as possible.